js/mexc3.js

```diff
@@ -670,7 +670,7 @@ module.exports = class mexc3 extends Exchange {
                 'strike': undefined,
                 'optionType': undefined,
                 'precision': {
-                    'amount': undefined,
+                    'amount': this.safeInteger (market, 'baseAssetPrecision'),
```

Comment:
Can you elaborate a bit on how this line fixes the amount in createSpotOrder ?

Reply:
because without it, when `amountToPrecision ()` is called in `createSpotOrder ()`, an undefined value is passed in 

```JavaScript
amountToPrecision (symbol, amount) {
        const market = this.market (symbol)
        return decimalToPrecision (amount, TRUNCATE, market.precision.amount, this.precisionMode, this.paddingMode)
    }
```
https://github.com/ccxt/ccxt/blob/master/js/base/Exchange.js#L1710

Reply:
Right, thanks!